src/include/rbd_types.h

```diff
@@ -74,6 +74,10 @@
 #define RBD_HEADER_SIGNATURE	"RBD"
 #define RBD_HEADER_VERSION	"001.005"
 
+#define GROUP_HEADER_PREFIX "group_header."
+
+#define GROUP_DIRECTORY "group_directory"
```

Comment:
Minor: not that anyone should be mixing pools, but can we put an "rbd_" prefix on this?